### DISCUSSION TITLE
Embedding python

### DISCUSSION THREAD

**Discussion by Uzair-M15** [pyscripter/python4delphi#479] [Q&A]
How would I go about embedding python to use with Python4Delphi. Ive added both the DLL Name and the DLL Path to the PythonEngine. But it doesnt seem to work

**Comment by pyscripter** [comment]
I am not sure what you want to do, but see the demos and the [Wiki pages](https://github.com/pyscripter/python4delphi/wiki). You can also use the [support forum](https://en.delphipraxis.net/forum/39-python4delphi/) for questions.

This [wiki page](https://github.com/pyscripter/python4delphi/wiki/FindingPython) may help.
